### PR TITLE
Logging resources selector

### PR DESCRIPTION
**Pull request by jinapurapu**
Adds CPU and memory selector fields to EditLogModal for both Logging and Logging DB, and displays the current value of these resource requests on the TenantLogging screen.

Screenshot from 2022-01-14 17-53-58
Screenshot from 2022-01-14 17-53-11

### CODE REVIEW

**Comment by jinapurapu** [comment]
Screenshot from 2022-01-20 09-49-06
Logging modal layout made consistent with Monitoring modal.

**Review by dvaldivia — CHANGES_REQUESTED** [review]
While trying to set a cpu 2 and memory 2 I get the following error
```
E: 2022/01/21 11:18:42 original error -> (operator_tenants.go:190: quantities must match the regular expression '^([+-]?[0-9.]+)([eEinumkKMGTP]*[-+]?[0-9]*)$')
```
Screen Shot 2022-01-21 at 11 18 52 AM

**Comment by cniackz** [comment]
While testing this branch today, I found the same problem Daniel found before:

Screen Shot 2022-01-26 at 7 10 12 PM

The error I got is the same I think:
`detailedMessage: "quantities must match the regular expression '^([+-]?[0-9.]+)([eEinumkKMGTP]*[-+]?[0-9]*)$'"
`

**Review by cniackz — CHANGES_REQUESTED** [review]
While testing this branch today, I found the same problem Daniel found before:

Screen Shot 2022-01-26 at 7 10 12 PM

The error I got is the same I think:
`detailedMessage: "quantities must match the regular expression '^([+-]?[0-9.]+)([eEinumkKMGTP]*[-+]?[0-9]*)$'"
`
And I am testing on the latest commit for this pull request:
```
commit 74fcc55c59676bb555327cefefe43b35c61749fe (HEAD -> logging-resources-selector)
Merge: e8c3329c 41b34645
Author: Prakash Senthil Vel <23444145+prakashsvmx@users.noreply.github.com>
Date:   Mon Jan 24 05:52:49 2022 +0000

    Merge branch 'master' into logging-resources-selector
```

**Review by cniackz — DISMISSED** [review]

File: operatorapi/operator_tenants.go [hunk 1]
```diff
@@ -1830,11 +1805,41 @@ func getTenantLogsResponse(session *models.Principal, params operator_api.GetTen
...
+		requestedDBCPU = strconv.FormatInt(requestedDBCPUQ.Value(), 10)
+		requestedDBMemQ := minTenant.Spec.Log.Db.Resources.Requests["memory"]
+		requestedDBMem = strconv.FormatInt(requestedDBMemQ.Value(), 10)
+
+		retval.LogCPURequest = requestedCPU
+		retval.LogMemRequest = requestedMem
+		retval.LogDBCPURequest = requestedDBCPU
+		retval.LogDBMemRequest = requestedDBMem
+	}
+	/*if minTenant.Spec.Log.Image == "" {
```
Comment (cniackz) [inline operatorapi/operator_tenants.go:1829]: Wonder if it is better to remove comments as this code is no longer used.

Comment (jinapurapu) [reply operatorapi/operator_tenants.go]: I removed the commented lines.

**Review by cniackz — APPROVED** [review]

**Review by dvaldivia — APPROVED** [review]

**Pull request merged**